sap/latest/install/index.md

```diff
@@ -67,8 +67,8 @@ Use the following steps to download the files required to install the SAP Connec
 
 1. Go to [Hyland Community](https://community.hyland.com/products/alfresco){:target="_blank"}, click **Product downloads**, and then download the SAP Connector distribution zip, which contains the following files:
 
-    * `sap-content-connector-repo-5.2.x.amp` for Content Services.
-    * `sap-content-connector-share-5.2.x.amp` for Alfresco Share.
+    * `sap-content-connector-repo-5.3.x.amp` for Content Services.
+    * `sap-content-connector-share-5.3.x.amp` for Alfresco Share.
     * `sap-content-connector-jco-packer-1.x.jar` - the **SAP JCo Packer tool** for merging the native SAP Java Connector libraries into the repository AMP file.
     * `sap-content-connector-encryptor-1.0.jar` to [encrypt plain-text passwords]({% link sap/latest/admin/reference.md %}#encryptpwd) for all SAP Connector related properties in the `alfresco-global.properties` file.
```

Comment:
```suggestion
    * `sap-content-connector-encryptor-1.x.jar` to [encrypt plain-text passwords]({% link sap/latest/admin/reference.md %}#encryptpwd) for all SAP Connector related properties in the `alfresco-global.properties` file.
```

Reply:
@psdfrisch Thanks for the review. I found a few more references to `sap-content-connector-encryptor-1.0.jar` - in `sap/latest/admin/reference.md`. Should I change these to `sap-content-connector-encryptor-1.x.jar` as well?

Reply:
Not that easy: in two cases it occurs in a command line, so using the real version 1.1 would be better, because the user can just copy and run it. On the other hand, in one of those lines the user would also have to exchange the "H3ll0W0rlD112"  pw string. So let's change it -1.x.jar - what do you think?

Reply:
Okay, that makes sense. With 5.3 only -1.1.jar is supplied. 
The pw string is used as an example in the command line in paragraph 4 - Create an encrypted password.